css/header.css

```
@@ -0,0 +1,102 @@
+.main__header {
+    background: url(/images/header_background.png);
+}
+
+.main__header .container {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+}
+
+.main__header .header__logo {
+    padding: 16px 10px;
+}
+
+.main__header .header__menu {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+}
+
+.main__header a {
+    color: var(--darkgray);
+}
+
+.header__menu nav > ul {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    column-gap: 10px;
+}
+
+.header__menu nav > ul > li {
+    position: relative;
+}
+
+.header__menu nav > ul > li > a {
+    padding: 12px 18px;    
+}
+
+.header__menu nav > ul > li:hover > a {
+    background-color: var(--gray);
+}
+
+.header__menu nav > ul > li:last-child > a {
+    background-color: var(--darkorange);
+    color: var(--white);
+    padding: 12px 16px;
+}
+
+.header__menu nav > ul > li:last-child:hover a {
+    background-color: var(--gray);
+    color: var(--darkgray)
+}
+
+.header__menu nav > ul > li:first-child > a::after,
+.header__menu nav > ul > li:nth-child(4) > a:after {
+    position: absolute;
+    content: '+';
+    margin-left: 5px;
+}
```

This can be achieved using `:has` selector. 

```suggestion
.header__menu nav > ul > li:has(ul) a::after {
    position: absolute;
    content: '+';
    margin-left: 5px;
}
```

However, almost most of the browsers have no support yet for this feature [https://caniuse.com/?search=has](https://caniuse.com/?search=has).

We can still achieve this by adding `content: '+';` to all elements `li a:after` by default. And if `li` has only one child which is `<a>`, which also means that it doesn't have submenus, we will set the content property to `content: '';`

```suggestion
.header__menu nav > ul > li a::after {
    position: absolute;
    content: '+';
    margin-left: 5px;
}

.header__menu nav > ul > li a:only-child::after {
    content: '';
}
```